docs/global.css

```diff
@@ -2,25 +2,9 @@
 @value colorsButtonBlue: #5e73e5;
 @value colorsButtonGreen: #00C37D;
 
-:global body .bp3-button {
-  font-weight: bold;
-  text-transform: uppercase;
-  background-color: #5e73e5;
-  padding: 8px 24px;
-  height: 40px;
-  margin-top: -8px;
-  border: none;
-  box-shadow: unset !important;
-  -webkit-box-shadow: unset !important;
-  border-radius: 4px;
-  background-image: unset !important;
-}
-
-:global body .bp3-button * {
-  color: #fff;
-  font-size: 12px;
+:global body {
+  overflow-y: scroll !important;
```

Comment:
Does this need `!important`? Placing this style on the `body` is not ideal either - would be nice to have it in some lower bit of html.

Reply:
I can't see an easy way to avoid this - it's purpose is for the search omnibar, which causes a big render jump when it opens because the scrollbar disappears. In practice, the entire site is scrolling on every page anyway.